blueocean-dashboard/src/main/js/creation/credentials/CredentialsApi.js

```diff
@@ -1,5 +1,14 @@
 import { capabilityAugmenter, Fetch, UrlConfig, Utils } from '@jenkins-cd/blueocean-core-js';
 
+const DOMAIN = 'blueocean-git-domain';
+const SCOPE = 'USER';
+
+function getCredentialsUrl() {
+    const path = UrlConfig.getJenkinsRootURL();
+    return Utils.cleanSlashes(`${path}/blue/rest/organizations/jenkins/credentials/user/`);
```

Comment:
@cliffmeyers not directly related to this PR but shouldn't base URI should not be hardcoded,  'blue' in /blue/rest/... could change. All of UI should be using appUrlBase set in blueocean-web/main.jsx, https://github.com/jenkinsci/blueocean-plugin/blob/master/blueocean-web/src/main/js/main.jsx#L143.